### PR TITLE
cloud/pkg/cloudhub/common/model: add unit test

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

**What type of PR is this?**

/kind test

**What this PR does / why we need it**:

Before changes:
```
ok  	github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/common/model	0.060s	coverage: 96.8% of statements
```

After changes:
```
ok  	github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/common/model	0.040s	coverage: 100.0% of statements
```

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by daixiang0** [comment]
need rebase

File: cloud/pkg/cloudhub/common/model/types_test.go [hunk 1]
```diff
@@ -195,3 +195,53 @@ func TestIsToEdge(t *testing.T) {
...
+		msg    model.Message
+		wanted string
+	}{
+		{
+			name:   "TestGetContent(): Case 1: empty content",
+			msg:    messageWithContent(nil),
+			wanted: "<nil>",
+		},
+		{
+			name:   "TestGetContent(): Case 1: content includes int",
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/common/model/types_test.go:225]:
```suggestion
			name:   "TestGetContent(): Case 2: content includes int",
```

Comment (daixiang0) [reply cloud/pkg/cloudhub/common/model/types_test.go]: Done.

**Pull request merged**